book/src/SUMMARY.md

```diff
@@ -16,6 +16,8 @@
       - [CSINode Object](csi-node-object.md)
     - [Features](features.md)
         - [Secrets & Credentials](secrets-and-credentials.md)
+            - [StorageClass (external-provisioner) Secrets](secrets-and-credentials-external-provisioner.md)
```

Comment:
I don't think we need to put the sidecar name in the title. It's an implementation detail.